### DISCUSSION TITLE
Amplify input volume more than 100%?

### DISCUSSION THREAD

**Discussion by shelbyKiraM** [ExistentialAudio/BlackHole#626] [Feature Requests]
This is supposedly possible with Audio Hijack, but the latest version doesn't work with macOS 13 beta while BlackHole DOES sooo... Any thoughts? I'm trying to get a mic that's always pretty quiet to be louder.

**Comment by devinroth** [comment]
First of all I think it’s a dumb idea as you will potentially damage your speakers.

But yes it’s possible. Route audio to BlackHole. Input from BlackHole to GarageBand. Slap on a limiter and boost your volume and output to your speakers.

Devin Roth

**Comment by shelbyKiraM** [comment]
It's more that my mic input is very quiet. I don't need louder speakers.

**Comment by devinroth** [comment]
I’m that case change the signal flow.

Mic to GarageBand. Add limiter. Output to BlackHole. Use BlackHole as input to Zoom etc.

Devin Roth

**Comment by shelbyKiraM** [comment]
Ahhh, GREAT point!! Installed, and I messed with the EQ to get better amplification than just turning up gain on the limiter could give me, but thanks for that! 👏🏼
